src/GafferArnold/IECoreArnoldPreview/Renderer.cpp

```diff
@@ -2616,6 +2616,13 @@ class ArnoldGlobals
 			}
 		}
 
+		typedef tbb::concurrent_unordered_map<std::string, IECoreScene::ConstCameraPtr> CameraMap;
+
+		const CameraMap &cameraMap()
```

Comment:
Why does this need to be public now? Can we keep encapsulation by moving `updateCameraMeshes()` to ArnoldGlobals? There are a lot of moving parts in this file, so minimising the coupling between each part is pretty worthwhile...